jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/PathMapper.kt

```diff
@@ -52,13 +52,13 @@ class PathMapper(private val mappings: List<PathMapping>) {
                 localPath
             }
 
-            return FileUtil.normalize(updatedPath)
+            return FileUtil.toCanonicalPath(updatedPath)
```

Comment:
Not sure how likely customers will have symlinks in the path. Do you think it is safer to use 
`toCanonicalPath(updatedPath, true)` instead as described [here](https://github.com/JetBrains/intellij-community/blob/master/platform/util/src/com/intellij/openapi/util/io/FileUtil.java#L633)?

Reply:
Couldn't hurt, also the integ tests are broken, so fixing them